daisy_workflows/image_import/windows/run_startup_scripts.cmd

```diff
@@ -33,9 +32,45 @@ for /f "tokens=2 delims=:" %%a in (
   netsh interface ipv4 set dnsservers "Local Area Connection 3" static address=%%a primary
 )
 
+echo "Translate: Setting timezone" > COM1:
 tzutil /s 'UTC'
 w32tm /resync
 
+echo "Translate: Verifying network connectivity" > COM1:
+REM Attempt to connect to metadata service and packages.cloud.google.com 5 times, taking as short as 5 seconds and up to 310 second.
+for /L %%I IN (1,1,5) do (
+  ping 169.254.169.254 -n 1
```

Comment:
Minor: consider extracting to a var/constant as it's reused in multiple places

Reply:
Good point. Moved to variables throughout the script and a double loop.